pkg/stats/collector.go

```diff
@@ -324,19 +371,10 @@ func (s *BufferedCollector) sendRuntimeStats() {
 	}()
 }
 
-func getBufferedCollectorArgs(c *config.Config) (processID string, opts []BufferedCollectorOpts) {
-	if c == nil {
-		return "", nil
+func HashMetricValue(s string) string {
+	if s == "" {
+		return s
 	}
-	var sender Sender
-	if c.GetStatsEnabled() && !strings.HasPrefix(version.Version, version.UnreleasedVersion) {
-		sender = NewHTTPSender(c.GetStatsAddress(), time.Now)
-	} else {
-		sender = NewDummySender()
-	}
-	return uuid.Must(uuid.NewUUID()).String(),
-		[]BufferedCollectorOpts{
-			WithSender(sender),
-			WithFlushInterval(c.GetStatsFlushInterval()),
-		}
+	v := xxhash.Sum64String(s)
+	return fmt.Sprintf("%02x", v)
```

Comment:
2 hex digits gives you 256 options which is considered safe usage for 16 unique examples. I don't think it's enough for branches, let alone commits which can accumulate quite heavily.

Reply:
This is full hex string with leading zero for 2 digit padding it doesn't cut the value